dpdata/unit.py

```diff
@@ -0,0 +1,169 @@
+from abc import ABC
+from scipy import constants
+
+AVOGADRO = constants.Avogadro                             # Avagadro constant
+ELE_CHG  = constants.elementary_charge                    # Elementary Charge, in C
+BOHR     = constants.value("atomic unit of length")       # Bohr, in m
+HARTREE  = constants.value("atomic unit of energy")       # Hartree, in Jole
+RYDBERG  = constants.Rydberg * constants.h * constants.c  # Rydberg, in Jole
```

Comment:
@amcadmus @njzjz  I've used constants from `scipy` package, which is the lastest